src/activities/align4-2players/Piece.qml

```diff
@@ -35,7 +35,7 @@ Image {
             name: "invisible"
             PropertyChanges {
                 target: piece
-                opacity: 0
+                opacity: 1
```

Comment:
Reverted all the changes.
Actually I was experimenting with this game, as I changed everything back to what it was, hence didn’t feel the need to check the difference, will take care of this from next time.